### ISSUE TITLE
How to deal the  invalid area of the depth?

### ISSUE THREAD

**Issue by mathmax12** [cogaplex-bts/bts#114]
Thanks for the amazing work and code.

When checking the depth map there are some invalid areas.
I wonder how you deal with this part during training, validation,  and test.

Thanks.

**Comment by cogaplex-bts** [comment]
@mathmax12 We simply ignore invalid pixels that do not have depth values. Please check our implementation. Thanks.

**Comment by mathmax12** [comment]
@cogaplex-bts 
Thanks. 
I got it.
Another question. in the paper "_**Because weights at
early convolutions are known to be well trained for primitive visual features, in the base networks, we fix the first two
convolutional layers as well as batch normalization parameters in our training.**_"
How to choose how much the first layers should we keep fixed?
In the paper, I saw the first two, and the batch norm is fixed.

Thanks

**Comment by mathmax12** [comment]
Another question is when using the nyu to training the model, for the loss calculation, the mask is generated with 
`if args.dataset == 'nyu':
        mask = depth_gt > 0.1`
I wonder how chose the 0.1 here.
Thanks.
